packages/components/src/range-control/tooltip.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Tooltip } from './styles/range-control-styles';
+
+const TOOLTIP_OFFSET_HEIGHT = 32;
+
+export default function SimpleTooltip( {
```

Comment:
We already have a Tooltip component, what's preventing us from using it instead?

Reply:
@youknowriad Haii! That's a good question :). I explained it a bit in the description. I'm happy to expand on it further if it helps.

Essentially, the positioning mechanism of the `Tooltip` component (aka. `Popover` under the hood) is visually laggy, as seen here:

https://d.pr/i/a6i3R4

(Note: in that demo, I used `Popover`, but the mechanism is the same)

Because of that, I opted to create a simple Tooltip just for this component. Granted, I don't think that's the ideal solution long term.

Hope this helps!